tests/functional/cylc-config/00-simple.t

```diff
@@ -52,10 +52,7 @@ cmp_ok "${TEST_NAME}.stderr" - </dev/null
 #-------------------------------------------------------------------------------
 TEST_NAME="${TEST_NAME_BASE}-section2"
 run_ok "${TEST_NAME}" cylc config -d --item=[runtime] "${WORKFLOW_NAME}"
-# Crude sorting to handle against change of dict order when new items added:
```

Comment:
This output is deterministic and should not be sorted, otherwise we're not checking that settings are in the right sections